tests/containers/podman_quadlet.pm

```diff
@@ -67,30 +103,53 @@ _EOF_
     # start the generator and check whether the files are generated
     systemctl("daemon-reload");
     check_unit_states();
-    systemctl("is-active sleeper.service", expect_false => 1);
-    systemctl("is-active sleeper-volume.service", expect_false => 1);
+    for my $unit (@units) {
+        systemctl("is-active $unit", expect_false => 1);
+    }
+
+    # start build unit
+    script_retry("podman pull $src_image", retry => 3, delay => 15, timeout => 180);
```

Comment:
```suggestion
    script_retry("podman pull $src_image", retry => 3, delay => 60, timeout => 180);
```
When retrying due to network issues, it is typically a good idea to have a delay of a minute or even longer.

If it fails now, the likelihood that it fails again immediately after is high. Let's give the network some time to heal itself, if that's possible.